cloudwatch_logs_common/include/cloudwatch_logs_common/log_service.h

```diff
@@ -52,11 +60,15 @@ class LogService : public Aws::CloudWatch::CloudWatchService<std::string, LogTyp
      * @param batcher used to batch / queue logs before publishing
      * @param file_upload_streamer used to save logs data and upload later in the event of network connectivity changes
      */
-  LogService(std::shared_ptr<Publisher<LogCollection>> log_publisher,
-             std::shared_ptr<DataBatcher<LogType>> log_batcher,
-             std::shared_ptr<Aws::FileManagement::FileUploadStreamer<LogCollection>> log_file_upload_streamer = nullptr)
-          : CloudWatchService(std::move(log_publisher), std::move(log_batcher)) {
-
+  LogService(
+    std::shared_ptr<Publisher<LogCollection>> log_publisher,
+    std::shared_ptr<DataBatcher<LogType>> log_batcher,
+    std::shared_ptr<Aws::FileManagement::FileUploadStreamer<LogCollection>> log_file_upload_streamer = nullptr
+  ) : CloudWatchService(
+    std::move(log_publisher),
+    std::move(log_batcher),
+    kLogEventMinimumPublishPeriod
```

Comment:
Default value?

Reply:
Sorry, not sure what you mean - this is a constant for CWLogs that we're passing in to the generic CW service, which defaults to 0.